README.md

```diff
@@ -7,28 +7,30 @@ This repository contains:
 - `nix/eval` - an incomplete Nix evaluator. It can't evaluate realistic Nix files, but it's a start.
 - `cmd/gon` - an utility that exposes these libraries from the command line.
 
-# How to build
+# Development
 
-```console
+## How to build
+
+```sh
```

Comment:
Actually `console` looks better because `shell` will highlight shell builtins, console will highlight lines starting with `$` and show lines without `$` (command output) in a different color.